config/cobbler/settings.yaml

```diff
@@ -388,17 +391,14 @@ nopxe_with_triggers: true
 # authentication within Cobbler Web and Cobbler XMLRPC.
 redhat_management_server: "xmlrpc.rhn.redhat.com"
 
-# if using authn_spacewalk in modules.conf to let Cobbler authenticate
-# against Satellite/Spacewalk's auth system, by default it will not allow per user
-# access into Cobbler Web and Cobbler XMLRPC.
-# in order to permit this, the following setting must be enabled HOWEVER
-# doing so will permit all Spacewalk/Satellite users of certain types to edit all
-# of Cobbler's configuration.
-# these roles are:  config_admin and org_admin
-# users should turn this on only if they want this behavior and
-# do not have a cross-multi-org seperation concern.  If you have
-# a single org in your satellite, it's probably safe to turn this
-# on and then you can use CobblerWeb alongside a Satellite install.
+# If using "authentication.spacewalk" for the key "modules.authentication.module" to let Cobbler authenticate against
+# Satellite/Spacewalk's auth system, by default it will not allow per user access into Cobbler Web and Cobbler XMLRPC.
```

Comment:
These comments are not easy to understand. What does it mean to "use X for the key in Y"?

Attempting to fix this:

```suggestion
# If you configure "authentication.spacewalk" in "modules.authentication.module" to let Cobbler authenticate against
# Satellite/Spacewalk's auth system, Cobbler does not allow user-based access into Cobbler Web and Cobbler XMLRPC by default.
```